prow.sh

```diff
@@ -1304,7 +1304,11 @@ main () {
             delete_cluster_inside_prow_job non-alpha
         fi
 
-        if tests_need_alpha_cluster && [ "${CSI_PROW_E2E_ALPHA_GATES}" ]; then
+        # If the cluster for alpha tests doesn't need any feature gates, then we
+        # could reuse the same cluster as for the other tests. But that would make
+        # the flow in this script harder and wouldn't help in practice because
+        # we have separate Prow jobs for alpha and non-alpha tests.
+        if tests_need_alpha_cluster; then
```

Comment:
if we have different prow jobs for alpha tests, is it going to recreate the cluster here? just want to make sure for each job, we only need to create cluster once..

Reply:
Each job will create and delete one cluster because either the one if path is taken or the other, but never both in the same job.